rpc_util.go

```diff
@@ -677,11 +698,36 @@ func recvAndDecompress(p *parser, s *transport.Stream, dc Decompressor, maxRecei
 	return d, nil
 }
 
+// Copied from go/src/io/ioutil/ioutil.go
```

Comment:
This could be problematic, as it will have legal (license) concerns.  At the least, we would need to add their license to our repo, which I would rather avoid.  Is there another way to implement this?